Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/PolDiffILLReduction.py

```diff
@@ -219,34 +239,131 @@ def PyInit(self):
 
         self.setPropertySettings('InstrumentCalibration', scan)
 
-    @staticmethod
-    def _normalise(ws):
-        """Normalises the provided WorkspaceGroup to the monitor 1."""
-        for entry in mtd[ws]:
-            mon = ws + '_mon'
-            ExtractMonitors(InputWorkspace=entry, DetectorWorkspace=entry,
-                            MonitorWorkspace=mon)
-            if 0 in mtd[mon].readY(0):
-                raise RuntimeError('Cannot normalise to monitor; monitor has 0 counts.')
-            else:
-                CreateSingleValuedWorkspace(DataValue=mtd[mon].readY(0)[0]/1000.0,
-                                            ErrorValue=np.sqrt(mtd[mon].readY(0)[0]/1000.0),
-                                            OutputWorkspace=mon)
-                Divide(LHSWorkspace=entry, RHSWorkspace=mon, OutputWorkspace=entry)
-                DeleteWorkspace(Workspace=mon)
-        return ws
+        self.declareProperty(name="NormaliseBy",
+                             defaultValue="Monitor",
+                             validator=StringListValidator(["Monitor", "Time"]),
```

Comment:
Do we need to have `None` as an option too? We do have in other techniques, it's useful sometimes when debugging.

Reply:
That can certainly be added, not a problem, and I do see the benefit for removing complexity during debugging reduction results.